src/theme/components/featuredContent.ts

```diff
@@ -46,37 +46,57 @@ const FeaturedContent = defineMultiStyleConfig({
             bgColor: "dark.ui.bg.default",
           },
           img: screenreaderOnly(),
+          containerType: "inline-size",
+          "@container (min-width: 0px)": {
+            " > div": {
```

Comment:
If possible, I'd like this to be similar to the `data-image` attributes and not rely so much on the DOM structure. Try to make the rules more general so that if the DOM gets updated, little needs to be updated on the CSS side.